reference/about-reference.md

```diff
@@ -1,59 +1,40 @@
 # The reference article
 
-## When do I need a reference article?
-
 Reference articles provide information-oriented descriptions of specific technology parts.
-
-The purpose of reference articles is to tell the facts about the technology as consistently and accurately as possible, without mixing procedural or instructional content.
+The purpose of reference articles is to describe the topic as consistently and accurately as possible, without mixing procedural or instructional content.
 
 Reference articles work well when they are:
 
-* consistent in structure, language and tone
-* contain descriptive information that is relevant to the reference topic's overview only
-* focus on information accuracy and the facts only.
+* Consistent in structure, language, and tone
+* Contain descriptive information that is relevant to the reference topic's overview
+* Focus on information accuracy and the facts
```

Comment:
MIssing a full stop to end the list?

Reply:
![image](https://user-images.githubusercontent.com/3914967/97099016-54b32500-16cf-11eb-9d12-e0f88766acef.png)